realm/realm-library/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -18,12 +18,21 @@
 
 import java.io.Closeable;
 import java.io.IOError;
+import java.util.concurrent.TimeUnit;
 
+import io.realm.exceptions.IncompatibleLockFileException;
+import io.realm.exceptions.RealmError;
 import io.realm.exceptions.RealmIOException;
 import io.realm.internal.async.BadVersionException;
+import io.realm.internal.log.RealmLog;
 
 public class SharedGroup implements Closeable {
 
+    // Keep these public so we can ask users to experiment with these values if needed.
+    // Should be locked down as soon as possible.
+    public static long[] INCREMENTAL_BACKOFF_MS = new long[] {1, 3, 10, 20}; // Will keep re-using last value until LIMIT is hit
+    public static long INCREMENTAL_BACKOFF_LIMIT_MS = 1000;
```

Comment:
Adjust to {1, 10, 20, 50, 100, 200, 400} and 3 sec timeout. This is a good comprise between being responsive and allowing the scheduler to work.